include/pmacc/memory/boxes/CachedBox.hpp

```diff
@@ -30,18 +30,18 @@ namespace pmacc
 {
     namespace CachedBox
     {
-        template<uint32_t Id_, typename ValueType_, class BlockDescription_, typename T_Worker>
+        template<uint32_t Id_, typename T_Mapping, typename ValueType_, class BlockDescription_, typename T_Worker>
```

Comment:
Only a note: we should call this template `T_MemoryMapping` to avoid being confused by the mapper we use to define where a kernel is executed. We use it everywhere in PMAcc and PIConGPU the template name `T_Mapper` is very near to `T_Mapping`

Reply:
Good point, will fix!